opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -982,13 +1014,18 @@ class EclMaterialLawManager
     std::vector<Opm::EclEpsScalingPointsInfo<Scalar>> unscaledEpsInfo_;
     OilWaterScalingInfoVector oilWaterScaledEpsInfoDrainage_;
 
-    Opm::EclMultiplexerApproach threePhaseApproach_;
+    GasOilScalingPointsVector gasOilUnscaledPointsVector_;
```

Comment:
They are satnum data, not cell data. If I understands the code correctly all partitions should create and store these vectors independently from the deck.